src/ontology/metadata/ordo.yml

```diff
@@ -71,3 +71,9 @@ comment: >
   recorded in ORDO. This is why great care needs to be taking interpreting diesease 2 gene relationships during ingest!
 base_prefix_map:
   Orphanet: http://www.orpha.net/ORDO/Orphanet_
+comments_about_this_source: >
```

Comment:
## `comments_about_this_source` in `ordo.yml`
As decided in #481. The relevance of this will kick in moreso when that issue is completed.